nixos/modules/services/misc/signald.nix

```diff
@@ -0,0 +1,102 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.signald;
+  dataDir = "/var/lib/signald";
+  defaultUser = "signald";
+in
+{
+  options.services.signald = {
+    enable = mkEnableOption ''
+      start the signald service with the socket at
+      /run/signald/signald.sock
+    '';
+
+    user = mkOption {
+      type = types.str;
+      default = defaultUser;
```

Comment:
I'm currently running it with `DynamicUser`. Why do we need a user like this here?